bgc_data/bgc_trip_metadata.sql

```diff
@@ -16,6 +16,8 @@ CREATE VIEW bgc_trip_metadata AS
     phytosampledepth_m,
     zoopsampledepth_m,
     biomass_mgm3 AS "Biomass_mgm3",
+    ashfreebiomass_mgm3 AS "AshFreeBiomass_mgm3", 
+    methods AS "Method",
```

Comment:
This column is not in the `bgc_trip` table yet, and only added to the schema in #31.
So I think it would be best to remove this commit from the current PR, and put it in a new one once the schema changes  in `bgc_trip` have been finalised.

Reply:
commit reverted